@xen-orchestra/xapi/vdi.mjs

```diff
@@ -76,6 +76,11 @@ class Vdi {
       })
     })
 
+    if (gm?.PV_drivers_detected) {
+      // It may happen that `PV_drivers_detected` returns `true` too quickly, and thus we destroy the cloudConfig VDI before the first boot
+      await new Promise(resolve => setTimeout(resolve, 1000 * 30))
+    }
+
```

Comment:
AFAIU, this code does not respect either the Plane card (unconditional 5 mins delay) nor the GitHub issue (only wait 30 secs in case of guest tools, 5 mins otherwise).